config/config.example.toml

```diff
@@ -298,6 +298,11 @@ lower_fetch_limit = 1800          # Lower limit for fetching entries from redis
 lock_key = "PRODUCER_LOCKING_KEY" # The following keys defines the producer lock that is created in redis with
 lock_ttl = 160                    # the ttl being the expiry (in seconds)
 
+[scheduler.server]
+port = 3000
+host = "127.0.0.1"
+workers = 1
```

Comment:
Would be preferable if you can add comments about the same here, and in the `deployment/scheduler/{consumer,producer}.toml` files.